### PR TITLE
Fix 22.06 style checks

### PR DESCRIPTION
**Pull request by mdemoret-nv**
Fixes style checks preventing #237 from passing

### CODE REVIEW

File: models/training-tuning-scripts/hammah-models/hammah-20211017-script.py [hunk 1]
```diff
@@ -1,38 +1,79 @@
...
+# limitations under the License.
 """
 Example Usage:
 python hammah-20211017-script.py \
        --trainingdata "../datasets/training-data/hammah-user123-training-data.csv"\
-       --valdata "../datasets/validation-data/hammah-user123-validation-data.csv" 
+       --valdata "../datasets/validation-data/hammah-user123-validation-data.csv"
 """
 
+import argparse
```
Comment (drobison00) [inline models/training-tuning-scripts/hammah-models/hammah-20211017-script.py:22]: why is this on a line by itself?

**Comment by mdemoret-nv** [comment]
@gpucibot merge

File: models/validation-inference-scripts/hammah-models/hammah-inference.py [hunk 2]
```diff
@@ -40,82 +44,86 @@ def date2min(time):
...
+        model = dill.load(f)
 
     scores = model.get_anomaly_score(X_val)
     X_val['ae_anomaly_score'] = scores
 
     X_val.sort_values('ae_anomaly_score', ascending=False).head(10)
-    #since inference is done, add the original columns back so the output will be the same as the input format
-    #X_val['ts_anomaly']=X_val_original['ts_anomaly']
+    # since inference is done, add the original columns back so the output will be the same as the input format
+    # X_val['ts_anomaly']=X_val_original['ts_anomaly']
```
Comment (drobison00) [inline models/validation-inference-scripts/hammah-models/hammah-inference.py:86]: delete?

File: models/validation-inference-scripts/hammah-models/hammah-inference.py [hunk 3]
```diff
@@ -40,82 +44,86 @@ def date2min(time):
...
     df = cudf.read_csv("hammah-user123-validation-data.csv")
     df = df.sort_values(by=['eventTime'])
     timearr = df.eventTime.to_array()
     START = stript(timearr[0])
-    END = stript(timearr[-1])
     timeobj = list(map(stript, timearr))
     hs = list(map(date2min, timeobj))
-    n, bins = cp.histogram(cp.array(hs), bins = cp.arange(0, max(hs)))
+    n, _ = cp.histogram(cp.array(hs), bins=cp.arange(0, max(hs)))
```
Comment (drobison00) [inline models/validation-inference-scripts/hammah-models/hammah-inference.py:93]: Maybe a nitpick, but I'd like to see all these stacked lines broken out a bit into blocks of logical steps or sub functions.

**Review by drobison00 — COMMENTED** [review]
Some suggestions/questions. Looks good overall.

**Pull request merged**